Resources/Prototypes/Entities/Structures/gates.yml

```diff
@@ -15,9 +20,10 @@
 
 - type: entity
   parent: BaseLogicItem
-  id: LogicGate
+  id: LogicGateOr
```

Comment:
This will probably break the construction graph

Reply:
the graph also has been changed